frameworks/elastic/docs/install.md

```diff
@@ -79,3 +79,49 @@ Elastic supports deployment on virtual networks on DC/OS (including the `dcos` o
 }
 ```
 As mentioned in the [developer guide](https://mesosphere.github.io/dcos-commons/developer-guide.html) once the service is deployed on a virtual network, it cannot be updated to use the host network.
+
+## TLS
+
+The Elastic service can be launched with TLS encryption. Enabling TLS will switch all internal communication between Elastic nodes to encrypted connections.
+
+Enabling TLS is possible only in `permissive` and `strict` cluster security modes. Both modes require a service account. Additionally, a service account must have the `dcos:superuser` permission. If the permission is missing the Elastic scheduler will not abe able to provision TLS artifacts.
```

Comment:
"is only possible". 

Also, in the json file further down, we explicitly mention Enterprise DC/OS. Should that be stated here too?